README.md

```diff
@@ -1,3 +1,39 @@
 # RHMAP Nagios for Docker
 
 Nagios server in a Docker container for RHMAP.
+
+# Setting up nagios service account
+
+In order to talk to the OpenShift and perform exec commands on pods, a service account will need to be created for use by nagios and given the proper permissions.
+
+1. Create the account using oc in the same project as the mbaas/core you are monitoring:
+
+```
+cat > nagios-sa.yaml << EOF
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nagios
+EOF
+ 
+oc create -f nagios-sa.yaml
+sudo oadm policy add-cluster-role-to-user cluster-reader system:serviceaccount:<rhmap-project>:nagios
```

Comment:
No `sudo`.

Instead, you should be logged in as a user with cluster-admin role.